staging/src/k8s.io/apiserver/pkg/endpoints/handlers/delete.go

```diff
@@ -198,6 +199,12 @@ func DeleteCollection(r rest.CollectionDeleter, checkBody bool, scope *RequestSc
 			return
 		}
 
+		if errs := metainternalversionvalidation.ValidateListOptions(&listOptions); len(errs) > 0 {
```

Comment:
Hmm, I'm wondering about this one - can this be done in one place instead everywhere where we parse it?

`DecoreParameters` comes to my mind actually...
@liggitt - WDYT?

Reply:
friendly ping